### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by SimonSchick**
I tried to update to the latests ecosystem dependencies, lint was easy enough, I'm unsure about the lack of the `secret` property into those 2 providers tho, I removed them to pass tests, please advise.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,8 +44,8 @@
     "boom": "7.x.x",
     "bounce": "1.x.x",
     "cryptiles": "4.x.x",
-    "hoek": "5.x.x",
-    "joi": "13.x.x",
+    "hoek": "^6.0.1",
```
Comment (AdriVanHoudt) [inline package.json]: Can you change these back to the .x.x notation?

**Comment by kanongil** [comment]
Yes, this the changed test behavior is caused by the `Hoek.deepEqual()` bugfix for https://github.com/hapijs/hoek/issues/260.

File: package.json (hunk 1 above)
Comment (SimonSchick) [reply package.json]: Ah yea, I keep forgetting about this, sry.

Comment (AdriVanHoudt) [reply package.json]: No problem, just ping me when ready 🎉

**Comment by SimonSchick** [comment]
AAAAAAAAAAAAAAAA

**Comment by SimonSchick** [comment]
all done

**Comment by SimonSchick** [comment]
boop

**Comment by hueniverse** [comment]
Merged manually.

**Pull request closed without merging**